### ISSUE TITLE
External link accessibility

### ISSUE THREAD

**Issue by kengdoj** [atbcb/usab-uswds#248]
For accessibility, to notify screen readers that the link is external, I suggest modifying the external-link.js to include `a.setAttribute("title", "external link")` so "external link" is announced after the link name.

```
if (nonGovDomain) {
        a.className = a.className + " " + "usa-link--external";
        a.setAttribute("title", "external link")
      }
```
If this is acceptable @apburnes , I suggest making this a fix for USWDS.

**Comment by bruce-usab** [comment]
Also, on my local build, I am getting the icon on every link (and not just non-.gov/.mil).  Not a big deal.  I am really glad we have this.  I am surprised it is not part of USWDS out of the box.

**Comment by kengdoj** [comment]
the federalist build of our site doesn't have the ext link on every link.

**Comment by bruce-usab** [comment]
Right, it is **_only_** my local build which has the defect.  It is a great feature for the production version!

**Comment by apburnes** [comment]
@kengdoj that's a great addition to me.  Thanks!

**Comment by kengdoj** [comment]
fixed for ab site in pr #254
